bin/spiced/src/lib.rs

```diff
@@ -68,14 +68,19 @@ pub struct Args {
 
 pub async fn run(args: Args) -> Result<()> {
     let current_dir = env::current_dir().unwrap_or(PathBuf::from("."));
-    let app = Arc::new(RwLock::new(
-        App::new(current_dir.clone()).context(UnableToConstructSpiceAppSnafu)?,
-    ));
     let auth = Arc::new(RwLock::new(runtime::load_auth_providers()));
     let df = Arc::new(RwLock::new(runtime::datafusion::DataFusion::new()));
     let pods_watcher = PodsWatcher::new(current_dir.clone());
+    let app = match App::new(current_dir.clone()).context(UnableToConstructSpiceAppSnafu) {
+        Ok(app) => Some(Arc::new(RwLock::new(app))),
+        Err(e) => {
```

Comment:
Wondering if we can just fallback to empty/default app instead of introducing the Option

Reply:
I like the Option, but as I noted in another comment we need to tweak it

Reply:
Thanks for the suggestions! I updated the code for tweaking the option, please help take a look if you are available.